src/pages/index.js

```diff
@@ -16,79 +22,241 @@ const cardListEl = document.querySelector(".cards__list");
 const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardFormElement = document.forms["add-card-form"];
 const cardSelector = "#card-template";
-
-/* validation */
-
+const profileImageEditButton = document.querySelector(
+  ".profile__image-container"
+);
+const changeProfilePictureModal = document.forms["change-profile-picture-form"];
+const profileImage = document.querySelector(".profile__image");
 const profileEditForm = document.forms["profile-form"];
-
-const editFormValidator = new FormValidator(
-  validationSettings,
-  profileEditForm
+const deleteCardForm = document.forms["delete-card-form"];
+const profileEditSubmitButton = document.querySelector(
+  "#profile-edit-submit-button"
 );
-const addFormValidator = new FormValidator(
-  validationSettings,
-  addCardFormElement
+const addCardModalSubmitButton = document.querySelector(
+  "#add-card-modal-submit-button"
+);
+const changeProfilePictureSubmitButton = document.querySelector(
+  "#change-profile-picture-submit-button"
 );
 
-addFormValidator.enableValidation();
-editFormValidator.enableValidation();
+const userInfo = new UserInfo({
+  titleSelector: ".profile__title",
+  descriptionSelector: ".profile__description",
+  profileImageSelector: ".profile__image",
+});
 
-function renderCard(cardData) {
-  const card = new Card(cardData, cardSelector, showPreviewImage);
+let section;
 
-  section.addItem(card.getView());
-}
+//api instantiate
 
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: renderCard,
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "75623fcf-ddee-44c9-84c0-daef735bad5c",
+    "Content-Type": "application/json",
   },
-  ".cards__list"
-);
+});
+
+//api on page loading
+
+api.getInitialCards().then((initialCards) => {
+  section = new Section(
+    {
+      items: initialCards,
+      renderer: renderCard,
+    },
+    ".cards__list"
+  );
+  section.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**